hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -463,134 +277,144 @@ fromLedgerTx (Ledger.Alonzo.ValidatedTx body wits isValid auxData) =
     Ledger.Alonzo.IsValid False ->
       TxScriptValidity TxScriptValiditySupportedInAlonzoEra ScriptInvalid
 
+-- | Show (compact) information about a Cardano transaction for debugging purpose.
 --
--- TxIn
---
-
-fromLedgerTxIn :: Ledger.TxIn Ledger.StandardCrypto -> TxIn
-fromLedgerTxIn = fromShelleyTxIn
-
-toLedgerTxIn :: TxIn -> Ledger.TxIn Ledger.StandardCrypto
-toLedgerTxIn = toShelleyTxIn
+-- NOTE: The function is 'incomplete' but can easily be extended to cover new
+-- needs.
+describeCardanoTx :: CardanoTx -> Text
+describeCardanoTx (Tx body _wits) =
```

Comment:
Didn't you also use the prefix `prettyXXX` for something like this in the past? i.e. `prettyTx` would work here.

Reply:
Yeah it's `prettyValue` and `prettyUtxo`already in this module -> rename this.